app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -0,0 +1,265 @@
+subheader_select: tutorials
+main_css_id: learn
+description: "Learn to create advanced JavaFX animations"
+last_update: 2024-05-17
+---
+
+The [javafx.animation](javafxdoc:AnimationPackageSummary) package offers a simple framework for creating animations and transitions in a JavaFX application.
+It operates on the principle of [WritableValue\<T\>](javafxdoc:WritableValue) which are used across JavaFX.
+It additionally provides a variety of built-in transitions for common effects, support for parallel and sequential transitions, and the ability to handle events upon animation completion.
+I will go through all types of animations, starting with `Animation` and its subclasses `Transition` and `Timeline`, before representing a lower level animation with `AnimationTimer`.
+While `Transition` provides a simpler and more user-friendly way to create animations, `Timeline` offers greater flexibility and is suitable for more complex animations.
+In contrast, `AnimationTimer` is designed for frame-by-frame updates and does not make use of `WritableValue<T>`.
+
+## WritableValue
+`WritableValue<T>` is an interface that wraps a value that can be read and set. It is commonly used for storing properties in JavaFX UI elements, like
+`width` or `height` in the [Rectangle](javafxdoc:Rectangle) shape.
+
+## Animation
+
+The [Animation](javafxdoc:Animation) abstract class provides the core functionality for `Transition` and `Timeline` animations.
+
+An `Animation` consists of multiple properties:
+- The `targetFramerate` is the maximum framerate (frames per second) at which this `Animation` will run.
+- The `currentTime` is the current point in time in the `Animation` as a [Duration](javafx:Duration).
+- The `rate` defines the direction and speed at which the `Animation` is expected to be played. It supports both positive and negative numbers.
+- The `cycleCount` defines the number of cycles of this `Animation`. It can't be changed while running and must be positive.
+- The `cycleDuration` is the `Duration` of one cycle of this `Animation`. It is the time it takes to play from start to end of the `Animation` **at the default rate of 1.0**.
+- The `totalDuration` indicates the total duration of this `Animation`, including repeats. It is the result of `cycleDuration * cycleCount` or possibly `Duration.INDEFINITE`.
+- The `delay` is the `Duration` that delays the `Animation` when starting.
+- The `autoReverse` property specifies whether the `Animation` will play in reverse direction on alternating cycles.
+- The `onFinished` event handler is used to define additional behavior when the `Animation` finished.
+- The `status` represents the current state of the `Animation`, possible states are `PAUSED`, `RUNNING` and `STOPPED`.
+
+Additionally, it provides several useful methods, like `play()`, `playFrom(String cuePoint)`, `pause()`, `stop()` and more to control the animations flow.
+A quick look into the JavaDocs provides a great overview of its functionalities.
```

Comment:
As discussed in the other thread, "the JavaDocs" needs to be changed. What about this?

```suggestion
A quick look into [its documentation](javafxdoc:Animation) provides a great overview of its functionalities.
```

(Yes, the same URL was already linked above, but when so explicitly mentioning the docs, why make the user reorient to find the right link?)